drivers/espi/espi_utils.h

```diff
@@ -20,7 +20,7 @@
  *
  * @return 0 on success, negative errno otherwise.
  */
-static int espi_manage_callback(sys_slist_t *callbacks,
+static inline int espi_manage_callback(sys_slist_t *callbacks,
```

Comment:
No, it won't since the larger CL used this function. But I noticed this PR cannot pass the "socketpair" test suite. The root cause is wrong function names for eSPI flash APIs. Hence, I added the fix and raised up the order of this CL. I'm also curious why the other eSPI driver CL won't encounter this problem?

Reply:
I think this should be the parent of your larger CL. It can stand on its own and fixes a potential problem. That way your larger eSPI commit won't have any test failures.

Reply:
Done.

Reply:
Done.